packages/rulesets/src/spectral/test/avoid-additional-properties.test.ts

```diff
@@ -0,0 +1,172 @@
+import { Spectral } from "@stoplight/spectral-core"
+import linterForRule from "./utils"
+
+let linter: Spectral
+const errorMessage = "The use of additionalProperties is not allowed except for user defined tags on tracked resources."
+beforeAll(async () => {
+  linter = await linterForRule("AvoidAdditionalProperties")
+  return linter
+})
+
+test("AvoidAdditionalProperties should find errors", () => {
```

Comment:
>AvoidAdditionalProperties [](http://example.com/codeflow?start=6&length=25)

Can u add a test for when tags is not directly under properties and check what the behavior is?

Reply:
Is there any other rule that would catch the existence of "tags" other than under properties?

Reply:
Added the test case

Reply:
tx for adding the test. I have a comment on improving the test case, pl take a look.